src/pool/srv_pool.c

```diff
@@ -5632,7 +5632,7 @@ ds_pool_replicas_update_handler(crt_rpc_t *rpc)
 
 	case POOL_REPLICAS_REMOVE:
 		rc = ds_rsvc_remove_replicas(DS_RSVC_CLASS_POOL, &id, ranks,
-					     &out->pmo_hint);
+					     true /* stop */, &out->pmo_hint);
```

Comment:
@liw And then here, we don't know that the rank failed, so we DO try to stop it before removal? (Compared to `replace_failed_replicas`)